explorer/src/components/account/AnchorAccountCard.tsx

```diff
@@ -0,0 +1,175 @@
+        decodedAnchorAccountData = accountDecoder.coder.accounts.decode(
+          layoutName,
+          accountBuffer
+        );
+      }
+    });
+  }
+
+  if (!foundAccountLayoutName || !decodedAnchorAccountData) {
+    return (
+      <ErrorCard
+        text={
+          "Failed to find matching anchor account type for account discriminator"
+        }
+      />
+    );
+  }
+
+  return (
+    <>
+      <div className="card">
+        <div className="card-header">
+          <div className="row align-items-center">
+            <div className="col">
+              <h3 className="card-header-title">{foundAccountLayoutName}</h3>
+            </div>
+          </div>
+        </div>
+
+        <div className="table-responsive mb-0">
+          <table className="table table-sm table-nowrap card-table">
+            <thead>
+              <tr>
+                <th className="w-1 text-muted">Key</th>
+                <th className="text-muted">Value</th>
+              </tr>
+            </thead>
+            <tbody className="list">
+              {decodedAnchorAccountData &&
+                Object.keys(decodedAnchorAccountData).map((key) => (
+                  <AccountRow
+                    key={key}
+                    valueName={key}
+                    // @ts-ignore
+                    value={decodedAnchorAccountData[key]}
+                  />
+                ))}
+            </tbody>
+          </table>
+        </div>
+        <div className="card-footer">
+          <div className="text-muted text-center">
+            {decodedAnchorAccountData &&
+            Object.keys(decodedAnchorAccountData).length > 0
+              ? `Decoded ${Object.keys(decodedAnchorAccountData).length} Items`
+              : "No decoded data"}
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function AccountRow({ valueName, value }: { valueName: string; value: any }) {
+  let displayValue: JSX.Element;
+  if (value && value.constructor && value.constructor.name === "PublicKey") {
+    displayValue = <Address pubkey={value} link />;
+  } else if (value && value.constructor && value.constructor.name === "BN") {
+    displayValue = <>{value.toString()}</>;
+  } else if (value && typeof value !== "object") {
+    displayValue = <>{String(value)}</>;
+  } else if (value) {
+    const displayObject = createDisplayObject(value);
+    displayValue = (
+      <ReactJson
+        src={JSON.parse(JSON.stringify(displayObject))}
+        collapsed={1}
+        theme="solarized"
+      />
+    );
+  } else {
+    displayValue = <>null</>;
+  }
+  return (
+    <tr>
+      <td className="w-1 text-monospace">{camelToUnderscore(valueName)}</td>
+      <td className="text-monospace">{displayValue}</td>
+    </tr>
+  );
+}
+
+function camelToUnderscore(key: string) {
+  var result = key.replace(/([A-Z])/g, " $1");
+  return result.split(" ").join("_").toLowerCase();
+}
+
+function createDisplayObject(object: Object): Object {
```

Comment:
I'm not clear on this function's purpose, or why its return value gets stringified and then immediately parsed, but I'll kick this can down the road since this review is already getting pretty long!

Reply:
Yeah I'm not sure how to best handle decoding anchor account data. The point of using a ReactJSON is to show nested anchor objects with a UI that supports expanding/collapsing, without having to create custom nested components. 

Unfortunately, just `stringify`-ing a nested object obfuscates PublicKeys & BigNums, which are pretty common (for example, the Creators array in the Candy Machine Account).

I'm very open to new ways of handling this

Reply:
Ohhhhh. I see. Yes! Please do rewrite this entire thing. Feel free to do it in a separate PR.

```
const obj = {
  myPubkey: PublicKey.default,
};
function replacer(key, value) {
  if (value instanceof PublicKey) {
    return value.toBase58();
  }
  return value;
}
console.log(JSON.stringify(obj, replacer));
> '{"myPubkey":"11111111111111111111111111111111"}'
```

Reply:
oh, wow, I feel dumb. Will do in follow-up PR